karate-tests/build-sandbox.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -x -e
```

Comment:
Not sure I understand the logic flow of this script.

Container is built, then running container checked for.
Would it be better to do this first before building the new container?
I'm guessing the freshly built container is run from somewhere else

Please clarify the logic behind this script